src/reconstruction/modelGeneration/test4HumanFctExt.m

```diff
@@ -239,201 +245,195 @@
     % %       FBA = optimizeCbModel(model,'max','zero');
     % %     TestSolution(k,1) = FBA.f);
     % %     TestSolutionName{k,1} = 'Human Recon 1 test mouse biomass';
-    % %  if ~isnan(TestSolution(k,1)); TestedRxns = [TestedRxns; model.rxns(find(abs(FBA.x)>tol))]; end ;k = k +1;clear FBA
-    %% "Human Recon 1 human  biomass"
+    % %  k = k +1;clear FBA
+  %% "Human Recon 1 human  biomass"
     model = modelOri;
-    model.c(find(model.c)) = 0;
     model.c(ismember(model.rxns,'biomass_reaction'))=1;
     if find(model.c)>0
         FBA = optimizeCbModel(model,'max','zero');
-        TestSolution(k,1) = FBA.f;
+        TestSolution(k,1) = FBA.f; TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)];
     else
         TestSolution(k,1) = NaN;
     end
     TestSolutionName{k,1} = 'Human Recon 1 test human biomass';
-    if ~isnan(TestSolution(k,1)); TestedRxns = [TestedRxns; model.rxns(find(abs(FBA.x)>tol))]; end ;k = k +1;clear FBA
+    k = k +1;clear FBA
     %% "Human Recon 1 human  biomass"
     model = modelOri;
-    model.c(find(model.c)) = 0;
     model.c(ismember(model.rxns,'biomass_maintenance_noTrTr'))=1;
     if find(model.c)>0
         FBA = optimizeCbModel(model,'max','zero');
-        TestSolution(k,1) = FBA.f;
+        TestSolution(k,1) = FBA.f; TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)];
     else
         TestSolution(k,1) = NaN;
     end
     TestSolutionName{k,1} = 'Human Recon 1 test human biomass (noTrTr)';
-    if ~isnan(TestSolution(k,1)); TestedRxns = [TestedRxns; model.rxns(find(abs(FBA.x)>tol))]; end ;k = k +1;clear FBA
+    k = k +1;clear FBA
     %% "Human Recon 1 human  biomass"
     model = modelOri;
-    model.c(find(model.c)) = 0;
+    
     model.c(ismember(model.rxns,'biomass_maintenance'))=1;
     if find(model.c)>0
         FBA = optimizeCbModel(model,'max','zero');
-        TestSolution(k,1) = FBA.f;
+        TestSolution(k,1) = FBA.f; TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)];
     else
         TestSolution(k,1) = NaN;
     end
     TestSolutionName{k,1} = 'Human Recon 1 test human biomass (maintenance)';
-    if ~isnan(TestSolution(k,1)); TestedRxns = [TestedRxns; model.rxns(find(abs(FBA.x)>tol))]; end ;k = k +1;clear FBA
+    if ~isnan(TestSolution(k,1)); TestedRxns = [TestedRxns; model.rxns(abs(FBA.x)>tol)]; end ;k = k +1;clear FBA
     %% do not apply base medium for Harvey
     if strcmp(test,'Harvey') == 0
-        model = modelOri;
+        model = modelOri; % What is this line doing?
```

Comment:
It seems like this is just a copy of a model. Eventually, may you change it such that only 1 variable name is used?

Reply:
Changed it here, the rest of the script uses the modelOri to avoid resetting c, and the exchange values each time.